autotest/gdrivers/gpkg.py

```diff
@@ -4298,3 +4298,35 @@ def test_gpkg_gti_gpkg_ext(tmp_vsimem):
     ds = gdal.Open(filename)
     assert ds.GetDriver().ShortName == "GPKG"
     assert ds.GetRasterBand(1).Checksum() == 4672
+
+
+###############################################################################
+# Test rename a raster table with SQL
+
+
+def test_gpkg_rename_raster_table(tmp_vsimem):
```

Comment:
you could parametrize with input data types gdal.GDT_Byte to test for the base 'tiles' case and gdal.GDT_UInt16 to test the '2d-gridded-coverage' case

Reply:
done and test added